content/doc/addons/metabase.md

```diff
@@ -68,6 +68,42 @@ Once you created your add-on, open the management URL or look for `MB_SITE_URL`
 
 * [Learn how to use Metabase](https://www.metabase.com/learn/metabase-basics/getting-started/find-data)
 
+## Password reset
+
+To be able to reset your password, you must have [set up an active SMTP server](#configuring-a-smtp-server) in the `e-mail` section of the administrator settings. You can also do it [using a Mailpace add-on](#using-a-mailpace-add-on). Once done, the forgot password procedure will ask you the user email address and send a reset link to it.
+
+If you don't have an active SMTP server configured, there is a manual procedure to get the reset link:
+
+{{% steps %}}
+
+### Identify the Java application of your Metabase add-on
+
+Its name is the same as the add-on name, followed by the add-on ID.
+
+### Go to its environment variables page and add the password reset command
+
+Add a `CC_PRE_RUN_HOOK` environment variable with this value:
+
+```
+MB_DB_CONNECTION_URI="$POSTGRESQL_ADDON_URI" java --add-opens java.base/java.nio=ALL-UNNAMED -jar metabase.jar reset-password email@example.com
```

Comment:
Maybe we should state that `email@example.com` must be replaced by the email of an actual user of the Metabase instance?